core/lib/protobuf_config/src/proto/eth_watch.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+
+package zksync.config;
+
+message ETHWatch {
+  optional uint64 confirmations_for_eth_event = 1; // optional
+  optional uint64 eth_node_poll_interval = 2; // required; ms
```

Comment:
`_ms` suffix is missing

Reply:
https://github.com/matter-labs/zksync-era/blob/ec6e923ddb5063d391c59e04dad28f9d2c15dcd1/core/lib/config/src/configs/eth_watch.rs#L13